src/Grid.cpp

```diff
@@ -241,30 +241,30 @@ void Grid::Output_RRDF_csv(const Simdat& sim, const string name){
 void Grid::Output_RRDF_bin(const Simdat& sim, const string name){
 	
 	// Make file name
-	const string binFile = sim.files.dataDir + "/" + sim.files.name + "." + name + ".bin";
+	const string binFile = sim.files.dataDir + "/" + sim.files.name + "." + name + ".mdf";
```

Comment:
Somewhat nitpicky comment, mostly just to document the decision making process. What is the `.mdf` file extension as compared to a more general `.dat` or `.bin` binary file?

Reply:
Random extension. ".bin" is more appropriate